include_core/omrhookable.h

```diff
@@ -76,7 +76,7 @@ typedef struct J9HookInterface {
 #define J9HOOK_AGENTID_LAST  ((uintptr_t)-1)
 
 /* time threshold (=100 milliseconds) for triggering the tracepoint  */
-#define OMRHOOK_DEFAULT_THRESHOLD_IN_MILLISECONDS_WARNING_CALLBACK_ELAPSED_TIME	100
+#define OMRHOOK_DEFAULT_THRESHOLD_IN_MICROSECONDS_WARNING_CALLBACK_ELAPSED_TIME	100*1000
```

Comment:
in this form please add spaces around `*` and add wrapping brackets: `100 * 1000`